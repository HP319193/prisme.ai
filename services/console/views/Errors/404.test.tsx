import Error404 from './404';
import renderer from 'react-test-renderer';

it('should render', () => {
  const root = renderer.create(<Error404 />);
  expect(root.toJSON()).toMatchSnapshot();
});

it('should render with a link', () => {
  const root = renderer.create(<Error404 link="https://prisme.ai" />);
  expect(root.toJSON()).toMatchSnapshot();
});

it('should render with a reason', () => {
  const root = renderer.create(<Error404 reason="why not" />);
  expect(root.toJSON()).toMatchSnapshot();
});
