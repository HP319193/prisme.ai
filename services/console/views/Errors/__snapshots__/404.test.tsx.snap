// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<a
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onTouchStart={[Function]}
>
  404
</a>
`;

exports[`should render with a link 1`] = `
<a
  href="https://prisme.ai"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onTouchStart={[Function]}
>
  404
</a>
`;

exports[`should render with a reason 1`] = `
<a
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onTouchStart={[Function]}
>
  why not
</a>
`;
